pgsql/pc_pgsql.h

```diff
@@ -116,6 +116,9 @@ char* pc_patch_to_hexwkb(const PCPATCH *patch);
 /** Returns OGC WKB for envelope of PCPATCH */
 uint8_t* pc_patch_to_geometry_wkb_envelope(const SERIALIZED_PATCH *pa, const PCSCHEMA *schema, size_t *wkbsize);
 
+/** Returns OGC WKB for bounding diagonal of PCPATCH */
+uint8_t* pc_patch_to_geometry_wkb_bounding_diagonal(const PCPATCH *pa, const PCSCHEMA *schema, size_t *wkbsize);
+
```

Comment:
I would rather have this function (and  `pc_patch_to_geometry_wkb_envelope`, see [this commit](https://github.com/LI3DS/pointcloud/pull/59/commits/788dc496ab536959997784bed37d495e3154d21d)) in the lib, so that it can be tested with make check and valgrind more easily.